controller/work_item_type_group.go

```diff
@@ -0,0 +1,51 @@
+package controller
+
+import (
+	"github.com/fabric8-services/fabric8-wit/app"
+	"github.com/fabric8-services/fabric8-wit/application"
+	"github.com/fabric8-services/fabric8-wit/workitem/typegroup"
+	"github.com/goadesign/goa"
+)
+
+// WorkItemTypeGroupController implements the work_item_type_group resource.
+type WorkItemTypeGroupController struct {
+	*goa.Controller
+	db application.DB
+}
+
+const APIWorkItemTypeGroups = "workitemtypegroups"
+
+// NewWorkItemTypeGroupController creates a work_item_type_group controller.
+func NewWorkItemTypeGroupController(service *goa.Service, db application.DB) *WorkItemTypeGroupController {
+	return &WorkItemTypeGroupController{
+		Controller: service.NewController("WorkItemTypeGroupController"),
+		db:         db,
+	}
+}
+
+// List runs the list action.
+func (c *WorkItemTypeGroupController) List(ctx *app.ListWorkItemTypeGroupContext) error {
+	res := &app.WorkItemTypeGroupSigleSingle{}
```

Comment:
done in https://github.com/fabric8-services/fabric8-wit/pull/1541/commits/99ef9aa45291ba855d0326bb58b40503258aea54

Thanks !